apps/studio/data/fetchers.ts

```diff
@@ -7,7 +7,6 @@ import { ResponseError } from 'types'
 import type { paths } from './api' // generated from openapi-typescript
 
 const DEFAULT_HEADERS = {
-  'Content-Type': 'application/json',
```

Comment:
will this cause issues for other fetch requests? Maybe better to use a regular `fetch` for deployFunction so it doesn't use DEFAULT_HEADERS.

Reply:
It's not an issue, `openapi-fetch` sends `application/json` by default when `bodySerializer` is not used. [Docs](https://openapi-ts.dev/openapi-fetch/api#bodyserializer)